test/fixtures/pdf_regexes.yml

```diff
@@ -0,0 +1,7 @@
+awesome_bank_pdf_regex:
```

Comment:
when testing in development you can use this regexes + the transactions.pdf located in the fixtures folder